.changelog

```diff
@@ -4,6 +4,8 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Fixed Python version number in VM Installation chapter of Installation and
+  Upgrade Guide
```

Comment:
```suggestion
- Removed sle-module-pythonX in VM Installation chapter of Installation and
  Upgrade Guide because SUSE Manager 4.3 does not require it
```